EarlGrey/EarlGrey.h

```diff
@@ -66,6 +66,7 @@
 #import <EarlGrey/GREYScrollActionError.h>
 #import <EarlGrey/GREYSyncAPI.h>
 #import <EarlGrey/GREYUIThreadExecutor.h>
+#import <EarlGrey/GREYTestHelper.h>
```

Comment:
should be sorted.

#import <EarlGrey/GREYTestHelper.h>
#import <EarlGrey/GREYUIThreadExecutor.h>